src/main/python/covata/delta/api.py

```diff
@@ -0,0 +1,83 @@
+#   Copyright 2016 Covata Limited or its affiliates
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from __future__ import absolute_import
+
+import json
+from abc import ABCMeta, abstractmethod
+import six
+
+import requests
+
+
+@six.add_metaclass(ABCMeta)
+class ApiClient(object):
+
+    DELTA_URL = 'https://delta.covata.cc/master'    # type: str
```

Comment:
yeah absolutely. usually we can use plain vanilla config.ini to have our config done without hassle. Unless it's super complex, the configparser may suffice. https://docs.python.org/2/library/configparser.html